File: lua/refactoring/tests/refactor/106/go/extract-block/extract_block.expected.go
package main

import "fmt"

func foo_bar(a int) {
    var test int = 1
    test_other := 1
    for idx := test - 1; idx < test_other; idx++ {
        fmt.Println(idx, a)
    }
}

func simple_function(a int) {
    foo_bar(a)
}

File: lua/refactoring/tests/refactor/106/php/extract-block/extract_block.expected.php
<?php

namespace Module;

function foo_bar (
    $a
) {
    $test = 1;
    $test_other = 11;
    for ($idx = $test - 1; $idx < $test_other; ++$idx) {
        print $idx;
        print $a;
    }
}

function simple_function($a)
{
    foo_bar($a);
}
